src/main/resources/errors.yaml

```diff
@@ -337,33 +340,33 @@ request-errors:
       ${SNIPPET.RESEND_USING_ONLY_DEFINED_COLUMNS}
 
   - scope: UPDATE
-    code: UPDATE_PRIMARY_KEY_COLUMNS
-    title: Cannot update on primary key columns
+    code: UNSUPPORTED_UPDATE_FOR_PRIMARY_KEY_COLUMNS
+    title: Primary key columns cannot be updated
     body: |-
       Cannot update primary key columns: ${updateOnPrimaryKeyColumns}.
       
       The table ${keyspace}.${table} defines the primary keys: ${primaryKeys}.
 
   - scope: UPDATE
-    code: ZERO_UPDATE_OPERATIONS_FOR_TABLE
-    title: Cannot update with zero operations
+    code: MISSING_UPDATE_OPERATIONS
+    title: Update operation requires at least one operation
```

Comment:
Should it be "Update command requires..." (otherwise "operation requires operation")